bigquery_magics/bigquery.py

```diff
@@ -467,10 +467,9 @@ def _query_with_bigframes(query: str, params: List[Any], args: Any):
         raise ValueError("Dry run is not supported by bigframes engine.")
 
     if bpd is None:
-        raise ValueError("Bigframes package is not installed.")
-
-    bpd.options.bigquery.project = context.project
-    bpd.options.bigquery.credentials = context.credentials
+        raise ValueError(
+            "Please install the 'bigframes' package (pip install bigframes) to use the bigframes engine."
+        )
```

Comment:
Let's keep this intact for now.

```suggestion
        )

    bpd.options.bigquery.project = context.project
    bpd.options.bigquery.credentials = context.credentials
```

At some point we will want to refactor this logic to use the new `magics_settings`, but not in this PR.